tools/generator/test/Platform+ExtensionsTests.swift

```diff
@@ -7,37 +7,36 @@ import XCTest
 extension PlatformExtensionsTests {
     func test_compatibility_osMismatch() throws {
         let platform = Platform.device(os: .tvOS)
-        XCTAssertEqual(platform.compatibility(with: other), .osNotEqual)
+        XCTAssertEqual(platform.compatibility(with: iOSarm64Platform), .osNotEqual)
     }
 
     func test_compatibility_variantMismatch() throws {
         let platform = Platform(
-            os: other.os,
+            os: iOSarm64Platform.os,
             variant: .tvOSDevice,
```

Comment:
The variant doesn't match now.

Reply:
It never did. The `other` was always `iOS`. For this test, I want `platform` to be identical except for the variant.

Reply:
I mean that os has to be a subset of variant. If `os` is something like `.iOS`, then `variant` has to then be `.iOSDevice` or `.iOSSimulator`, otherwise the data is garbage.

Reply:
I switched it to `.iOSSimulator`.